keyboards/evyd13/gud70/readme.md

```diff
@@ -0,0 +1,17 @@
+# Gud70
+
+A replacement PCB for the Uniqey C70.
+
+* Keyboard Maintainer: [Evyd13](https://github.com/evyd13)
+* Hardware Supported: Gud70 PCB
+* Hardware Availability: [Open source on GitHub](https://github.com/evyd13/gud70-pcb)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?